NitroxServer/Communication/LiteNetLib/PortCheckerSupport.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Net;
+using LiteNetLib.Layers;
+using LiteNetLib;
+using System.Collections.Generic;
+using LiteNetLib.Utils;
+using System.Net.Sockets;
+namespace NitroxServer.Communication.LiteNetLib;
+
+public class PortCheckerSupport : PacketLayerBase
+{
+    public static bool active;
+    public NetManager netManager;
+    public PortCheckerSupport() : base(0)
+    {
+        active = false;
+    }
+    
+    public override void ProcessInboundPacket(ref IPEndPoint endPoint, ref byte[] data, ref int length)
+    {
+        if (active)
+        {
+            Log.Info("Incoming packet");
+            netManager.SendUnconnectedMessage(data, endPoint);
+        }
```

Comment:
This will respond back to all packets received and becomes noise when players are joining the server. So I recommend following changes:

 - Add warning that port checker must be disabled to allow players to join.
 - Change packet `data` so LiteNetLib will ignore it (Set `length` to 0?), send original back to port checker server (using `UdpClient`).

Reply:
Implemented